frontend/src/app.jsx

```diff
@@ -16,7 +14,7 @@ export default function App() {
       setTasks(tasks);
     }
     getTasks();
-  }, [update]);
+  }, []);
```

Comment:
Removed state variable from the dependency array as explained [here](https://github.com/nds-fsd/Abd-ha-todo-app/pull/3/files#r1505501386).